data.js

```diff
@@ -74,29 +73,55 @@ exports.data = [
             label: "wwf",
         },
         description: {
-            content: "The WWF is active since 1961 and therefore a well experienced player within the environmental engagement game. The biggest Pro on WWF is that it is well known and People trust the organization. They do not have a precise focus on the climate crises but there is also a focus on it. One of their goals for 2030 is to reduce greenhouse gas emissions by 40%.",
-            updatedBy: "Andre", updatedAt: "2019-08-18"
+            content: "The World Wide Fond for nature (WWF) is active since 1961 and therefore a well experienced player within the environmental engagement game. One of their goals for 2030 is to reduce greenhouse gas emissions by 40%.",
+            updatedBy: "Andre", updatedAt: "2020-01-21"
         },
         established: {
             content: "[1961](http://wwf.panda.org/knowledge_hub/history/)",
             updatedBy: "Andre", updatedAt: "2019-08-06"
         },
         kpis: {
-            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press Mentions",
-            updatedBy: "Andre", updatedAt: "2019-08-06",
-
+            content: "- Actions done  \n- Participants in actions\n- Members\n- Press mentions \n- Advised organization",
+            updatedBy: "Andre", updatedAt: "2020-01-21",
         },
+        actions: {
+            content: "There is a big variety. Here are some examples:"+
+                    "With their expertise as a big environmental player they advise (countries, communities and private companies)[https://www.worldwildlife.org/initiatives/climate] in the topic."+
```

Comment:
Needs a space in "examples:With"

Reply:
Same issues in the next 5 lines. Also newlines `\n`s are missing.

Reply:
done